build/rollup-plugin-clean-once.ts

```diff
@@ -39,7 +39,7 @@ export default function rollupPluginCleanOnce(): Plugin {
       const rootDir = process.cwd();
       const dirToDelete = options.dir
         ? path.resolve(rootDir, options.dir)
-        : path.dirname(path.resolve(rootDir, options.file));
+        : path.dirname(path.resolve(rootDir, options.file!));
```

Comment:
What does the ! do here? I suspect I'm showing my TS ignorance.

Reply:
Had to look it up too, tells TS that this is a non-null value. (`string | null` => `string`)

Reply:
Yes, I was less rigorous with this since it was part of the build process and not the extension itself. We could be even stricter if we wanted, but I'm not sure the value is there.

Reply:
It's important to note that `!` is just stripped from the code. I've adding a few of them around this codebase, especially where we're wrapping things in `try` anyway. They honestly aren't great, but this PR was already becoming huge. I could do a followup if updating our config to disallow them and address the remaining issues.